resource_processor/vmss_porter/runner.py

```diff
@@ -168,13 +169,25 @@ async def invoke_porter_action(msg_body: dict, sb_client: ServiceBusClient, mess
 
     # Handle command output
     if returncode != 0:
-        error_message = "Error context message = " + " ".join(err.split('\n')) + " ; Command executed: ".join(porter_command)
-        resource_request_message = service_bus_message_generator(msg_body, statuses.failed_status_string_for[action], error_message)
+        error_message = "Error message: " + " ".join(err.split('\n')) + "; Command executed: " + " ".join(porter_command)
+
+        pass_despite_of_error = False
+        if "uninstall" == action and "could not find installation" in err:
```

Comment:
What happens in the scenario where the install has failed part way through - are Azure resources left behind?

Reply:
That is handled today correctly - TF state includes the provisioned resources and when porter will try to uninstall it will be handled else where and won't reach this block of code as there is an installation